scc/tests/args.rs

```diff
@@ -1,20 +1,31 @@
 use std::path::PathBuf;
 
+use rstest::rstest;
+use rstest_reuse::{self, *};
 use scc::opts::*;
 use winsplit;
 
 pub fn test_fix_args(arg_str: &str, expected_args: &[&str]) -> Result<Vec<String>, String> {
     let args = fix_args(winsplit::split(arg_str));
     (args == expected_args.iter().map(|&s| s.into()).collect::<Vec<String>>())
         .then_some(args)
-        .ok_or("Does not match expected output".to_owned())
+        .ok_or({
+            // println!("fixed: {:#?}", args);
+            // println!("expected: {:#?}", expected_args);
```

Comment:
leftovers?

Reply:
Are you ok with printing nicer debug info when a test fails? I can clean this up a bit if so, or just remove the commented-out stuff if not.